completions/ant

```diff
@@ -9,7 +9,7 @@ _ant_parse_targets()
     # parse buildfile for targets
     while read -rd '>' line; do
         if [[ $line =~ \<(target|extension-point)[[:space:]].*name=[\"\']([^\"\']+) ]]; then
-            targets+=" ${BASH_REMATCH[2]}"
+            ret+=" ${BASH_REMATCH[2]}"
```

Comment:
`ret` needs to be initialized in advance. Also, can we use arrays?

```suggestion
            ret+=("${BASH_REMATCH[2]}")
```

Reply:
Ah, sure, thanks!